tests/libres_tests/res/enkf/test_es_update.py

```diff
@@ -205,9 +206,13 @@ def test_that_posterior_has_lower_variance_than_prior_with_init_files(copy_case)
     df_default = GenKwCollector.loadAllGenKwData(ert, "default")
     df_target = GenKwCollector.loadAllGenKwData(ert, "target")
 
-    assert df_default["COEFFS:COEFF_A"].var() > df_target["COEFFS:COEFF_A"].var()
-    assert df_default["COEFFS:COEFF_B"].var() > df_target["COEFFS:COEFF_B"].var()
-    assert df_default["COEFFS:COEFF_C"].var() > df_target["COEFFS:COEFF_C"].var()
+    # We expect that ERT's update step lowers the "size" of the covariance
+    # of the parameters, where "size" can be measured by the determinant.
```

Comment:
How about using the concept of "generalized variance" here? https://en.wikipedia.org/wiki/Generalized_variance
E.g.
"We expect that ERT's update step lowers the generalized variance for the parameters"

Reply:
Much better, thanks.